### ISSUE TITLE
Seurat fails due to ggplot2 version incompatibility

### ISSUE THREAD

**Issue by lparsons** [galaxyproject/tools-iuc#3008]
New installs of the Seurat wrapper will get `ggplot2 3.3.0` which is not compatible with `seurat 3.1.2`. There was [a fix](https://github.com/satijalab/seurat/commit/14cfc8ca1b0f44c310bd1c9ceb7ac1f458c33f74) made to [Seurat in version 3.1.4](https://github.com/satijalab/seurat/tree/v3.1.4) to address this.

Suggest adding a requirement of `ggplot2 < 3.3.0` for this version of the tool wrapper as well as pushing a new version that includes `Seurat 3.1.5` (current version) with the fix for `ggplot2 3.3.0`.

```
|  Quitting from lines 122-126 (/tmp/tmpruo2td1h/job_working_directory/000/2/working/Seurat.spin.Rmd)
|  Error in data.frame(group = sort(x = group.use), x = x.divs) :
|  arguments imply differing number of rows: 273, 0
|  Calls: render ... withVisible -> eval -> eval -> <Anonymous> -> data.frame
```

**Comment by bgruening** [comment]
@lparsons this is probably better adressed in the conda package.

**Comment by lparsons** [comment]
> @lparsons this is probably better adressed in the conda package.

I don't disagree, and feel free to decline #3009, but please accept at least #3010. Mucking with old version of conda-forge packages is beyond my reach atm, esp. to fix a problem that maintains and old version. If we don't fix 3.1.2, I'll just have to uninstall from my Galaxy (rather than leave it broken) and just replace with 3.1.5.

**Comment by nsoranzo** [comment]
@lparsons I think you should be able to manually fix the conda environment for 3.1.2 instead of removing that tool version.

**Comment by nsoranzo** [comment]
Merged #3010, feel free to reopen if there's anything else we could do.
